packages/list/mwc-list-base.ts

```diff
@@ -29,11 +29,31 @@ import {Layoutable, ListItemBase, RequestSelectedDetail} from './mwc-list-item-b
 
 export {ActionDetail, createSetFromIndex, isEventMulti, isIndexSet, MWCListIndex, SelectedDetail} from './mwc-list-foundation';
 
+function debounceLayout(
+    callback: <T>(this: T, updateItems: boolean) => void, waitInMS = 50) {
```

Comment:
YouTube conformance issue basically boils down to please use a different argument name

Reply:
What argument? updateItems? callback?

Reply:
hahaha oops I meant `this`

Reply:
oh - `this`?
I've removed it - all tests pass, linting seems good :)